packages/video_player/video_player_web/analysis_options.yaml

```diff
@@ -0,0 +1,10 @@
+# This is a temporary file to allow us to land a new set of linter rules in a
```

Comment:
Let's add a TODO to link to https://github.com/flutter/flutter/issues/45440

Reply:
Agreed, but I think the TODO link should be for a bug that's specific to the inline `ignore`s being blanked and specific to this plugin. I think this is a separate issue than the existing bug because it's a new regression, not an existing failure from before the linter was introduced.

Reply:
Ah ok, I thought the issue is (at least partially) about removing all analysis_options.
We can make a different issue to fix this specific lint.

Reply:
I have created https://github.com/flutter/flutter/issues/55000 for the cleanup.